tests/index.test.js

```
@@ -0,0 +1,65 @@
+const request = require('supertest');
+const express = require('express');
+
+const request = require('supertest');
```

Biome (1.9.4)

[error] Shouldn't redeclare 'request'. Consider to delete it or rename it.

'request' is defined here:


(lint/suspicious/noRedeclare)